geps/gep-1897/index.md

```diff
@@ -238,35 +198,12 @@ Thus, the following additions would be made to the Gateway API:
-| HTTPRoute  | Listener tls               | No                                            | No                            |
-| HTTPRoute  | No listener tls            | No                                            | No                            |
-| TLSRoute   | Listener Mode: Passthrough | Yes                                           | No                            |
-| TLSRoute   | Listener Mode: Terminate   | Yes                                           | Implementation-dependent      |
-| TLSRoute   | Listener Mode: Passthrough | No                                            | No                            |
-| TLSRoute   | Listener Mode: Terminate   | No                                            | No                            |
-| TCPRoute   | Listener TLS               | Yes                                           | Implementation-dependent      |
-| TCPRoute   | No listener TLS            | Yes                                           | Implementation-dependent      |
-| TCPRoute   | Listener TLS               | No                                            | No                            |
-| TCPRoute   | No listener TLS            | No                                            | No                            |
-| UDPRoute   | Listener TLS               | Yes                                           | No                            |
-| UDPRoute   | No listener TLS            | Yes                                           | No                            |
-| UDPRoute   | Listener TLS               | No                                            | No                            |
-| UDPRoute   | No listener TLS            | No                                            | No                            |
-| GRPCRoute  | Listener TLS               | Yes                                           | Implementation-dependent      |
-| GRPCRoute  | No Listener TLS            | Yes                                           | Implementation-dependent      |
-| GRPCRoute  | Listener TLS               | No                                            | No                            |
-| GRPCRoute  | No Listener TLS            | No                                            | No                            |
+|Route has a BackendTLSPolicy| Backend is targeted by a BackendTLSPolicy?    | Connect to backend  with TLS? |
+|----------------------------|-----------------------------------------------|-------------------------------|
+|Mode `TLS`                  | Yes                                           | **Yes**                       |
+|Mode `None`                 | Yes                                           | No                            |
+|No                          | Yes                                           | **Yes**                       |
+|No                          | No                                            | No                            |
```

Comment:
I think you need to keep the original preamble to this diagram, and maybe just add another column to the original diagram.